draft-liu-teas-transport-network-slice-yang.txt

```diff
@@ -0,0 +1,1568 @@
+             +----------+-----------------------+-----------+
+             | nw       | ietf-network-topology | [RFC8345] |
+             +----------+-----------------------+-----------+
+             | tet      | ietf-te-topology      | [RFC8795] |
+             +----------+-----------------------+-----------+
+             | te-types | ietf-te-types         | [RFC8776] |
+             +----------+-----------------------+-----------+
+             | ns-topo  | ietf-ns-topo          | RFCXXXX   |
+             +----------+-----------------------+-----------+
+
+             Table 1: Prefixes and Corresponding YANG Modules
+
+   RFC Editor Note: Please replace XXXX with the RFC number assigned to
+   this document.  Please remove this note.
+
+2.  Modeling Considerations
+
+   An IETF network slice is modeled as network topology defined in
+   [RFC8345], with augmentations.  A new network type "network-slice" is
+   defined in this document.  When a network topology data instance
+   contains the network-slice network type, it represents an instance of
+   an IETF network slice topology.
+
+2.1.  Relationships to Related Topology Models
+
+   There are several related YANG data models that have been defined in
+   IETF.  Some of these are:
+
+   Network Topology Model: Defined in [RFC8345].
+
+
+
+Liu, et al.              Expires 13 August 2023                 [Page 4]
+
+Internet-Draft      Network Slice Topology Data Model      February 2023
+
+
+   Network Slicing Model: Defined in
+   [I-D.ietf-teas-ietf-network-slice-nbi-yang].
+
+   OTN Slicing: Defined in [I-D.ietf-ccamp-yang-otn-slicing].
+
+   Figure 1 shows the relationships among these models.  The box of
+   dotted lines denotes the model defined in this document.
+
+        +----------+                 +----------+
+        | Network  |                 | Network  |
+        | Slice    |                 | Topology +
+        | NBI YANG >------+          | Model    |
+        | Model    |      |          | RFC 8345 |
+        +----+-----+      |          +-----+----+
+             |            |                |
+             |augments    |references      |augments
+             |            |                |
+        +----^-----+      |          ......^.....
+        | OTN      |      +----------: Network  :
+        | Slicing  | augments        : Slice    :
+        | Model    >-----------------: Topology :
+        |          |                 : Model    :
+        +----------+                 ''''''''''''
+
+                       Figure 1: Model Relationships
+
+2.2.  ACTN for Network Slicing
+
+   Since ACTN topology data models are based on the network topology
+   model defined in [RFC8345], the augmentations defined in this
+   document are effective augmentations to the ACTN topology data
+   models, resulting in making the ACTN framework [RFC8453] and data
+   models [I-D.ietf-teas-actn-yang] capable of slicing networks with the
+   required network characteristics.
```

Comment:
Is this statement consistent with the proposition in draft-ietf-teas-applicability-actn-slicing-02

Reply:
It seems that the relationship still holds. One may combine any topology model augmented from the base network topology model (including the ACTN topology models) and NS topology model (multi-inheritance)